src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colSup = new ColorSupplier();
+
+    private Figure [] figure = {new Circle(new Random().nextInt(20) + 1, colSup.getRandomColor()),
+            new Square(new Random().nextInt(20) + 1, colSup.getRandomColor()),
+            new Triangle(new Random().nextInt(20) + 1, new Random().nextInt(20) + 1,
+                    colSup.getRandomColor()),
+            new Rectangle(new Random().nextInt(20) + 1, new Random().nextInt(20) + 1,
+                    colSup.getRandomColor()),
+            new Trapezoid(new Random().nextInt(20) + 1, new Random().nextInt(20) + 1,
+                    new Random().nextInt(20 + 1), colSup.getRandomColor())};
+
```

Comment:
let's use switch/case here, create a random number (from 0 to 3 and use default case for the last figure type) and depending on that number you'll use switch case to create a random figure.